clusterloader2/pkg/measurement/util/wait_for_conditions.go

```diff
@@ -116,7 +117,7 @@ func WaitForGenericK8sObjects(ctx context.Context, dynamicClient dynamic.Interfa
 
 // countObjectsMatchingConditions counts objects that have a successful or failed condition.
 // Function assumes the conditions it looks for are mutually exclusive.
-func countObjectsMatchingConditions(objectConditions [][]metav1.Condition, successfulConditions []string, failedConditions []string) (successful int, failed int, all int) {
+func countObjectsMatchingConditions(objectsWithConditions []ObjectSimplification, successfulConditions []string, failedConditions []string) (successful []string, failed []string, all int) {
```

Comment:
Nit: consider renaming `all` to `count` (and this function usage in `WaitForGenericK8sObjects`) - `all` suggests some collection while `count` (or `total`) implies it's a number.

Reply:
done